### PR TITLE
AvalancheGo 1.9.3

### PR DESCRIPTION
**Pull request by aaronbuchwald**
This PR adds documentation for the chain aliases flag, which should go in the next AvalancheGo release v1.9.3.

### CODE REVIEW

File: docs/nodes/maintain/avalanchego-config-flags.md [hunk 1]
```diff
@@ -152,6 +152,24 @@ Specifies the directory that contains chain configs, as described [here](chain-c
...
+
+```json
+{
+  "2K33xS9AyP9oCDiHYKVrHe7F54h2La5D8erpTChaAhdzeSu2RX": [
+    "Swimmer"
+  ]
+}
+```
+
+The above example aliases the Blockchain whose ID is `"2K33xS9AyP9oCDiHYKVrHe7F54h2La5D8erpTChaAhdzeSu2RX"` to `"Swimmer"`. Chain aliases are added after adding primary network aliases and before any changes to the aliases via the admin API. This means that the first alias included for a Blockchain on a Subnet will be treated as the `"Primary Alias"` instead of the full blockchainID. The Primary Alias is used in all metrics and logs.
```
Comment (yulin-dong) [inline docs/nodes/maintain/avalanchego-config-flags.md:167]: Can you clarify `Chain aliases are added after adding primary network aliases and before any changes to the aliases via the admin API.`?

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: docs/nodes/maintain/avalanchego-config-flags.md (hunk 1 above)
Comment (yulin-dong) [reply docs/nodes/maintain/avalanchego-config-flags.md:167]: @aaronbuchwald ^^

File: docs/nodes/maintain/avalanchego-config-flags.md [hunk 2]
```diff
@@ -667,7 +685,13 @@ If this node is a validator, when a container is inserted into consensus, send a
 
 #### `snow-mixed-query-num-push-non-vdr` (uint):
 
-fmt.Sprintf("If this node is not a validator, when a container is inserted into consensus, send a Push Query to %s validators and a Pull Query to the others. Must be <= k. Defaults to `0`.
```
Comment (StephenButtolph) [inline docs/nodes/maintain/avalanchego-config-flags.md:670]: oof

File: docs/nodes/maintain/subnet-configs.md [hunk 3]
```diff
@@ -38,6 +38,12 @@ Subnet validators can choose not to publish contents of blockchains via this con
 This is a node-specific configuration. Every validator of this Subnet has to use this configuration in order to create a full private Subnet.
 :::
 
+#### `proposerMinBlockDelay` (time.Duration):
+
+The minimum delay enforce in building snowman++ blocks. Default is set to 1 second.
```
Comment (StephenButtolph) [inline docs/nodes/maintain/subnet-configs.md:43]:
```suggestion
The minimum delay performed when building snowman++ blocks. Default is set to 1 second.
```

File: docs/nodes/maintain/subnet-configs.md [hunk 4]
```diff
@@ -38,6 +38,12 @@ Subnet validators can choose not to publish contents of blockchains via this con
...
+
+As one of the ways to control network congestion, Snowman++ makes sure that no two blocks can be build before a `proposerMinBlockDelay` seconds delay. Some high-performance custom VM may find this too strict. This flag allows tuning of maximal frequency at which blocks can be built.
```
Comment (StephenButtolph) [inline docs/nodes/maintain/subnet-configs.md:45]:
```suggestion
As one of the ways to control network congestion, Snowman++ will only build a block `proposerMinBlockDelay` after the parent block's timestamp. Some high-performance custom VM may find this too strict. This flag allows tuning the frequency at which blocks are built.
```

**Review by yulin-dong — APPROVED** [review]

**Pull request merged**